src/web/app/pages-instructor/student-profile/student-profile.component.html

```diff
@@ -4,7 +4,7 @@
     <div class="col-12">
       <div class="row">
         <div class="col-md-2 col-3">
-          <img src="{{getPictureUrl(studentProfile?.pictureKey)}}" class="profile-pic float-right">
+          <img src="{{getPictureUrl(studentProfile.pictureKey)}}" class="profile-pic float-right">
```

Comment:
You should use `(error) => ()` handler to update the picture to `/assets/images/profile_picture_default.png`?

The current logic is a bit weird and there will be no more pictureKey returning for studentProfile.

Reply:
Yes you are right, I have made it better.